packages/artifact/src/internal/download-http-client.ts

```diff
@@ -112,10 +146,20 @@ export class DownloadHttpClient {
     artifactLocation: string,
     downloadPath: string
   ): Promise<void> {
```

Comment:
Nit: this method is big enough that I'd prefer it to be its own class, but that may be the C# developer in me.  Not needed for this PR.